src/tokens/mainnet.json

```diff
@@ -566,6 +566,19 @@
         "0xEc901DA9c68E90798BbBb74c11406A32A70652C3"
       ]
     },
+    {
+      "address": "0xe454C546C8F3875e928910Abc653De5f8c432F11",
+      "decimals": 18,
+      "mintUrl": "https://app.kodiak.finance/#/liquidity/pools/0xe454C546C8F3875e928910Abc653De5f8c432F11?chain=berachain_mainnet",
+      "name": "BIXBT-WBERA",
```

Comment:
Please also add the `BIXBT` token (`0x147c9a7f5265d7ca9a671c3817b212a0cf88856e`)

Reply:
just added sir

Reply:
Your push didn't go through. I suggest you rebase from my changes.

Reply:
Ohh yes! I've updated all commits. Is it ok now?